repo2docker/buildpacks/conda/install-miniconda.bash

```diff
@@ -43,9 +43,12 @@ if [[ "${CONDA_VERSION}" != "${MINICONDA_VERSION}" ]]; then
     conda install -yq conda==${CONDA_VERSION}
 fi
 
+# Install Mamba and symlink it to conda
```

Comment:
I think we're no longer doing the "and symlink it to conda" step (which I agree with!)